ludwig/features/text_feature.py

```diff
@@ -99,43 +148,26 @@ def feature_meta(column, preprocessing_parameters: PreprocessingConfigDict, back
             ngram_size=preprocessing_parameters["ngram_size"],
             compute_idf=preprocessing_parameters["compute_idf"],
             processor=backend.df_engine,
+            prompt_template=prompt_template,
         )
-
-    @staticmethod
-    def get_feature_meta(
-        column, preprocessing_parameters: PreprocessingConfigDict, backend, is_input_feature: bool
-    ) -> FeatureMetadataDict:
-        vocabulary = TextFeatureMixin.feature_meta(column, preprocessing_parameters, backend)
+        # Note: The vocabulary's max_sequence_length includes the prompt template, which is merged into the column prior
+        # to computing feature metadata.
         logger.info(
-            f"Max length of feature '{column.name}': {vocabulary.line_length_max} (without start and stop symbols)"
+            f"Max length of feature '{column.name}': {vocabulary.max_sequence_length} (without start and stop symbols)"
         )
 
-        # Use sequence_length if provided, otherwise use max length found in dataset.
-        if preprocessing_parameters["sequence_length"] is not None:
-            logger.info(
-                f"Setting max length to sequence_length={preprocessing_parameters['sequence_length']} provided in "
-                f"preprocessing parameters"
+        max_sequence_length, max_sequence_length_99ptile = _get_metadata_reconciled_max_sequence_length(
+            preprocessing_parameters, vocabulary
+        )
+
+        if is_input_feature and max_sequence_length < vocabulary.prompt_template_num_tokens:
+            raise ValueError(
+                f"The input feature's max sequence length ({max_sequence_length}) is shorter than the prompt template "
+                f"length ({vocabulary.prompt_template_num_tokens}). This will truncate all unique information. "
+                "Consider making the template shorter or increasing the input feature's max sequence length."
```

Comment:
nit: I think same as before, would be good to be explicit in what exact change to make, something like set your input feature's max_sequence_length to a value >> {vocabulary.prompt_template_num_tokens}. 

Not entirely sure what the best wording would be or if this is too redundant in terms of info in the ValueError message, but wanted to call it out anyway